tools/clang/tools/dxlib-sample/dxlib_sample.cpp

```diff
@@ -226,20 +231,23 @@ class LibCacheManager {
   std::shared_mutex m_mutex;
 };
 
+static hash_code CombineWStr(hash_code hash, LPCWSTR Arg) {
+  CW2A pUtf8Arg(Arg, CP_UTF8);
+  unsigned length = strlen(pUtf8Arg.m_psz);
```

Comment:
Consider using wcslen and not doing the conversion - the hash shouldn't care about the text contents anyhow.